packages/kitchen-sink/src/index.ts

```diff
@@ -0,0 +1,10 @@
+export * from '@rise-tools/kit'
+export * from '@rise-tools/kit-expo-router'
+export * from '@rise-tools/kit-forms'
+export * from '@rise-tools/kit-haptics'
+export * from '@rise-tools/kit-linking'
+export * from '@rise-tools/kit-lucide-icons'
+export * from '@rise-tools/kit-qrcode'
+export * from '@rise-tools/kit-svg'
+export * from '@rise-tools/kit-tamagui-toast'
+export * from '@rise-tools/tamagui'
```

Comment:
I think we may want to rename `tamagui` to `kit-tamagui`?

Reply:
yes I agree, lets do that